build-docs.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * build-html.javascript
```

Comment:
You forgot to change the name :smile:

Reply:
good catch. fixed in follow-on commit face42de.